portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,32 @@
 /**
  * Adds a random greeting to the page.
```

Comment:
Comment talks about a greeting, yet the function adds a fact (or replaces it if one already exists).